app/controllers/forest_liana/application_controller.rb

```diff
@@ -31,6 +31,31 @@ def forest_user
       @jwt_decoded_token
     end
 
+    def define_belongsTo_id(records, already_mapped_records = {})
```

Comment:
This function allow to add a `{ data: { id: 1 } }` on all belongsTo which are not retrieved from the previous query. For that we just instantiate an empty object with an id so that the serializer put the data